bot/util/utilities.py

```diff
@@ -198,6 +268,64 @@ def generate_features_string(features):
     return str_features
 
 
+def get_text_channel(channel_id: str, guild: discord.Guild) -> discord.TextChannel:
+    """Parses a message id string and searches the text channel in the passed guild
+
+    Args:
+        channel_id (str): The id of the channel (might also be surrounded by '<#' and '>'
+        guild (discord.Guild): The guild in which the channel is searched.
+
+    Returns:
+        discord.Channel: The found channel.
+
+    Raises:
+        ValueError: If the chanel could not be found.
+    """
+    try:
+        if channel_id.startswith('<#'):
+            channel_id = channel_id[2:-1]
+        channel = guild.get_channel(int(channel_id))
+    except ValueError:
+        raise ValueError('Channel to post embed to could not be found. Use channel ID or channel name (linked #)')
+
+    if channel is None:
+        raise ValueError('Channel to post embed to could not be found. Use channel ID or channel name (linked #)')
+
+    return channel
+
+
+def is_pastebin_link(json_string: str) -> bool:
```

Comment:
Yes, makes code easier to understand and adheres to single responsibility principle.
We should generally make more functions to abstract logic away.

Reply:
If we can hide non self-explanatory one- liners (or 2,3,4...-liners) behind trivially understandable function names it will improve the code by a lot.

Also I read a good measure is, if a function has more than 20 lines, usually it is long enough to be split up.
(except complex Algorithms ofc)